README.md

```diff
@@ -823,22 +823,30 @@ When transferring TOML files over the internet, the appropriate MIME type is
 Comparison with Other Formats
 -----------------------------
 
-In some ways TOML is very similar to JSON: simple, well-specified, and
-maps easily to ubiquitous data types. JSON is great for serializing
-data that will mostly be read and written by computer programs. Where
-TOML differs from JSON is its emphasis on being easy for humans to
-read and write. Comments are a good example: they serve no purpose
-when data is being sent from one program to another, but are very
-helpful in a configuration file that may be edited by hand.
-
-The YAML format is oriented towards configuration files just like
-TOML. For many purposes, however, YAML is an overly complex
-solution. TOML aims for simplicity, a goal which is not apparent in
-the YAML specification: http://www.yaml.org/spec/1.2/spec.html
-
-The INI format is also frequently used for configuration files. The
-format is not standardized, however, and usually does not handle more
-than one or two levels of nesting.
+TOML shares traits with other file formats used for application configuration
+and data serialization, such as YAML and JSON. TOML and JSON both are simple and
+use ubiquitous data types, making them easy to code for or parse with machines.
+TOML and YAML both emphasize human readability features, like comments that make
+it easier to understand the purpose of a given line. TOML differs in combining
+these, rather than permitting the much more complex syntax available in YAML or
+the rat's nests of curly braces that can happen in JSON.
+
+Because TOML is explicitly intended as a configuration file format, parsing it
+is easy, but it is not intended for serializing arbitrary data structures. TOML
+always has a hash table at the top level of the file, which can easily have data
+nested inside its keys, but it doesn't permit top-level arrays or floats, so it
+cannot directly serialize some data. There is also no standard identifying the
+start or end of a TOML file, which can complicate sending it through a stream.
+These details must be negotiated on the application layer.
```

Comment:
Well, are you saying, instead, that the rest of the paragraph is sufficient? Because I'm not so sure, given that people keep raising the issue (and so you wind up repeating it in the issues, frequently).

In addition I feel that sentence strengthens the conclusion of the paragraph in terms of writing flow.

Reply:
FWIW, this is nit-picking and I don't feel strongly either way; let's go with your preference here. :)